### ISSUE TITLE
My doubts about the experimental results

### ISSUE THREAD

**Issue by KamiEbieSan** [yuli0103/CGB-DM#7]
I retrained the model using the vanilla code under all constraints, but I found that some metrics were lower than those reported in the paper, especially the unds metric under the c and cwh conditions: 0.918 for c and 0.854 for cwh .Could this be due to the hyperparameter settings of the code or some other reason? Thank you.

**Comment by yuli0103** [comment]
Thank you for your interest. Due to other parallel work in progress, we plan to update the codebase after the paper is accepted. The current code is an older version.

**Comment by KamiEbieSan** [comment]
Thanks
It seems that the salbox normalization part in the dataloader did not work correctly
`sal_box[::2] /= width
sal_box[1::2] /= height`
It seems that it should replaced by the following code，because sal_box[1::2]  is an tensor
`sal_box[:, [0, 2]] /= width
sal_box[:, [1, 3]] /= height`

**Comment by KamiEbieSan** [comment]
Thanks
It seems that the salbox normalization part in the dataloader did not work correctly
```sal_box[::2] /= width 
   sal_box[1::2] /= height```

It seems that it should replaced by the following code，because sal_box[1::2] is an tensor

```sal_box[:, [0, 2]] /= width 
   sal_box[:, [1, 3]] /= height
   ```

**Comment by KamiEbieSan** [comment]
Thanks
It seems that the salbox normalization part in the dataloader did not work correctly
```sal_box[::2] /= width
   sal_box[1::2] /= height```
It seems that it should replaced by the following code，because sal_box[1::2] is an tensor
```sal_box[:, [0, 2]] /= width 
   sal_box[:, [1, 3]] /= height```

**Comment by KamiEbieSan** [comment]
Thanks
It seems that the salbox normalization part in the dataloader did not work correctly
```
sal_box[::2] /= width
sal_box[1::2] /= height
```
I think it should replaced by the following code，because sal_box[1::2] is an empty tensor
```
sal_box[:, [0, 2]] /= width 
sal_box[:, [1, 3]] /= height
```

**Comment by KamiEbieSan** [comment]
Thanks
It seems that the salbox normalization part in the dataloader did not work correctly
```
sal_box[::2] /= width
sal_box[1::2] /= height
```
I think it should be replaced by the following code，because sal_box[1::2] is an empty tensor
```
sal_box[:, [0, 2]] /= width 
sal_box[:, [1, 3]] /= height
```

**Comment by SpadgerBoy** [comment]
> I retrained the model using the vanilla code under all constraints, but I found that some metrics were lower than those reported in the paper, especially the unds metric under the c and cwh conditions: 0.918 for c and 0.854 for cwh .Could this be due to the hyperparameter settings of the code or some other reason? Thank you.

The metric of unds for cwh measured by me is also around 0.85 . May I ask if you have found what problem caused it, whether it is hyperparameters or other issues?

**Comment by SpadgerBoy** [comment]
> I retrained the model using the vanilla code under all constraints, but I found that some metrics were lower than those reported in the paper, especially the unds metric under the c and cwh conditions: 0.918 for c and 0.854 for cwh .Could this be due to the hyperparameter settings of the code or some other reason? Thank you.

The metric of unds for cwh measured by me is also around 0.85 . May I ask if you have found what problem caused it, whether it is hyperparameters or other issues?
